tools/topology/sof-cml-rt5682-kwd.m4

```diff
@@ -29,6 +29,9 @@ ifdef(`HSEARPROC', , `define(HSEARPROC, volume)')
 ifdef(`DMICPROC', , `define(DMICPROC, passthrough)')
 ifdef(`DMIC16KPROC', , `define(DMIC16KPROC, passthrough)')
 
+# FIXME: Using DMIC16kHz instead of DMIC16k, otherwise M4 does not return.
+define(DMIC_16k_PCM_NAME, DMIC16kHz)
```

Comment:
@plbossart why do we need this at all? the reason you're running into issues is likely that fact that pipe-kfbm-capture.m4 already adds a PCM called DMIC16k.

Reply:
if I don't have this then the .conf file has a 'PCM_NAME' difference with the original, the macro is not defined and is taken as text.